python/deadcode/return.py

```diff
@@ -1,6 +1,5 @@
 
-# ruleid: return-not-in-function
```

Comment:
instead of commenting the code out, you can changes this to # todoruleid:  return-not-in-function